### ISSUE TITLE
Network property is in germany set to road even a network is available

### ISSUE THREAD

**Issue by rene-mueller** [openmaptiles/planetiler-openmaptiles#143]
Network property is in Germany set to `road` even a network is available. The streets should always have a network so that the streets can be equipped with the national signs, as with mapbox street style.

US:
https://onthegomap.github.io/planetiler-demo/#14.14/43.67017/-92.86298
![grafik](https://github.com/openmaptiles/planetiler-openmaptiles/assets/48084558/35612724-47bf-419a-bde5-8452a2abebcd)

Germany:
https://onthegomap.github.io/planetiler-demo/#17.01/50.365394/11.785406
![grafik](https://github.com/openmaptiles/planetiler-openmaptiles/assets/48084558/3eea3fac-d288-481e-a2f6-52c5c2904feb)

**Comment by msbarry** [comment]
I think this is a limitation of openmaptiles. It only defines a small number of network types: https://github.com/openmaptiles/openmaptiles/blob/master/layers/transportation/network_type.sql. Projects like [openstreetmap-americana](https://github.com/ZeLonewolf/openstreetmap-americana) use the `route_n` tags on those features to render highway shields since a road may belong to multiple networks.

**Comment by rene-mueller** [comment]
@msbarry Ok, then from my point of view it would make sense to include this as a shield property in transportation_name.

**Comment by msbarry** [comment]
Yeah definitely, you can open it up as an issue in openmaptiles then if/when they adopt we'll port to planetiler-openmaptiles. Any deviation from standard openmaptiles in this repo needs to be hidden behind a command line flag you opt into.

**Comment by rene-mueller** [comment]
@msbarry that sounds logical. i open an issue there. :)
